Docker/build/WDQS/entrypoint.sh

```diff
@@ -13,7 +13,7 @@ done
 
 set -eu
 
-export BLAZEGRAPH_OPTS="-DwikibaseHost=${WIKIBASE_HOST}"
+export BLAZEGRAPH_OPTS="${BLAZEGRAPH_OPTS} -DwikibaseHost=${WIKIBASE_HOST}"
```

Comment:
Perhaps BLAZEGRAPH_OPTS should be after `-DwikibaseHost=${WIKIBASE_HOST}`
I believe this will allow people that make use of `BLAZEGRAPH_OPTS` via the docker image to also override the `-DwikibaseHost=` setting if they desire.

Reply:
Otherwise / instead, we could perhaps rename this env var to be extra blazegraph opts?

Reply:
I don't have a strong opinion on this, so `EXTRA_BLAZEGRAPH_OPTS` works for me.